src/main/java/tc/oc/pgm/Config.java

```diff
@@ -384,4 +384,19 @@ public int getDestroyMatchDelay() {
       return destroyMatchDelay;
     }
   }
+
+  public static class SidebarMessage {
+    public static boolean enabled() {
+      return getConfiguration().getBoolean("sidebar-message.enabled", false);
+    }
+
+    public static String format() {
+      return ChatColor.translateAlternateColorCodes(
+          '&', getConfiguration().getString("sidebar-message.text"));
+    }
+
+    public static boolean atTop() {
+      return getConfiguration().getBoolean("sidebar-message.top", false);
+    }
```

Comment:
Would maybe be easier to have message-top and message-bottom, with the ability to leave either of them blank?
That way you convert 3 settings into 2 (remove enabled & atTop, keep message-top and message-bottom), it gives more flexibility for people trying to configure it, and it's (maybe?) easier to code the condition.